components/canvas/Node.module.scss

```diff
@@ -67,15 +67,24 @@
   padding: 12px 22.5px 12px 22.5px;
 }
 
-.node__role-container {
+.node__info-container {
   height: 13px;
   padding: 5px 10px;
+}
+
+.node__info-text {
+  white-space: nowrap;
+  text-overflow: ellipsis;
+  overflow: hidden;
+}
```

Comment:
I think this class should be extended to all text elements on a card containing user generated content including description which I believe is the most important text to truncate as the current limit of 1000 characters is far too high for the ellipsis to be displayed.

Reply:
CSS does not have a native way of applying text-overflow styling on vertical overflows, so this would not work on description.